src/admin-views/editor/fieldset/price.php

```diff
@@ -4,19 +4,25 @@
 	$post_id = get_the_ID();
 }
 $validation_attrs = [
-	'data-validation-error="' . esc_attr( sprintf( _x( '%s price must be greater than zero.', 'ticket price validation error', 'event-tickets' ), tribe_get_ticket_label_singular( 'ticket_price_validation_error' ) ) ) . '"'
+	'data-validation-error="' . esc_attr( sprintf(
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 1b3ce3ea08b3a40bda58ec1cdd4fc161</sub>